docs/Technical-Documentation/user_role_management.md

```diff
@@ -147,11 +147,11 @@ data_prepper.user_set.add(user)
 
 3.) After moving into the `tdpapp` directory, the admin will then have to set the alias for the python executable if it has not been set and execute the shell script to promote the existing user.
```

Comment:
@jtwillis92 @abottoms-coder, do the steps below need to be updated? I walked through these a couple of weeks ago and noticed at least line 152 seemed outdated?